src/tablo/plugins/autosize/autosizeGrid.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import t, { maybe } from 'tcomb';
+import { props } from 'tcomb-react';
+import { pure, skinnable } from 'revenge';
+import cx from 'classnames';
+import FlexView from 'react-flexview';
+import ResizeSensor from '../../../resize-sensor/ResizeSensor';
+
+export default (Grid) =>
+
+  @props({
+    className: maybe(t.String),
+    autosize: maybe(t.Boolean)
+  }, { strict: false })
```

Comment:
we can safely use `{ strict: false }` as additional props will be checked by `Tablo` component